lib/mimdb/movies/movies_genres.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Mimdb.Movies.MoviesGenres do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "movies_genres" do
+    field :movie_id, :id
+    field :genre_id, :id
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(movies_genres, attrs) do
+    movies_genres
+    |> cast(attrs, [])
```

Comment:
If we are not casting any parameters, it makes me think we are not using this changeset. Can this be removed or is there a bug here?

Reply:
I ended up removing the cast